src/pages/home/ReportScreen.js

```diff
@@ -38,13 +37,14 @@ const ReportScreen = (props) => {
                 onNavigationMenuButtonClicked={() => Navigation.navigate(ROUTES.HOME)}
             />
             <View style={[styles.dFlex, styles.flex1]}>
-                <ReportView reportID={activeReportID} />
+                <ReportView
+                    reportID={activeReportID}
+                />
             </View>
         </ScreenWrapper>
     );
 };
 
-ReportScreen.displayName = 'ReportScreen';
```

Comment:
Since you ended up changing this back to a functional component, we do need the `displayName` property as per our react style guide:

> When writing a stateless component you must ALWAYS add a displayName property and give it the same value as the name of the component (this is so it appears properly in the React dev tools)

Since none of the changes left in this file are necessary, I might just revert them all, but NAB.